agent/pkg/database/main.go

```diff
@@ -50,12 +46,14 @@ func CreateEntry(entry *tapApi.MizuEntry) {
 	GetEntriesTable().Create(entry)
 }
 
-func initDataBase(databasePath string) *gorm.DB {
-	temp, _ := gorm.Open(sqlite.Open(databasePath), &gorm.Config{
+func InitDataBase(databasePath string) *gorm.DB {
```

Comment:
This file is going to be [deleted](https://github.com/up9inc/mizu/pull/279/files#diff-63458fb030016b15ab78b2b9748f9030a1338b49cc951ed43a830c6350fb974f). So this PR shouldn't change this file.

https://github.com/up9inc/mizu/pull/427 should wait https://github.com/up9inc/mizu/pull/279 to be merged.

Reply:
We should know how to deal with conflicts, we can't block ready to merge PR because of future PRs

Reply:
@IgorGov how is https://github.com/up9inc/mizu/pull/279 a future PR? Drafted on Sep 17, marked as ready 6 days ago. I also requested a review of @RamiBerm 3 days ago. Which is even before https://github.com/up9inc/mizu/pull/427 is opened.